game/core/src/main/java/com/gg/game/Model/Positionable.java

```diff
@@ -1,7 +1,17 @@
 package com.gg.game.Model;
 
-public interface Positionable {
-    void setPos(int x, int y);
-    void setPos(Vector position);
-    Vector getPos();
-}
+abstract class Positionable {
```

Comment:
Föredrar delegering mer än arv.


```suggestion
interface Positionable {
    void setPos(int x, int y);
    void setPos(Vector position);
    Vector getPos();
}

class Decor implements positionable{
Vector vector = ....

@Override
parentMethod(){
vector.doStuff();
}
```

Reply:
Yeah I agree. The more I sat with it the more I like that idea. Will change that up before I make another PR.